contributing/release-schedule.md

```diff
@@ -1,84 +1,154 @@
+
+The **stable** release is built with the _production_ configuration and served to most AMP traffic. Since the **beta** release is also built from the _production_ configuration, it represents the exact build which will become **stable** the following week (ignoring the possibility of [cherry-picks](#cherry-picks)).
+
+An additional **control** release channel exists, and is also served to 0.5% of AMP traffic. This release contains the same binaries as **stable**, and is used as a traffic-eqivalent baseline to compare **experimental** and **beta** against for metrics and error reporting. It cannot be opted into.
+
+See the [release cadence section](#release-cadence) for a more detailed breakdown of the release promotion cadence. To determine if a PR has been included in the current _weekly_ release, look for the GitHub labels _PR Use: In Canary_ or _PR Use: In Production_.
+
+### LTS
+
+The **lts** release channel provides a previous **stable** build for four-month intervals. Every four weeks, the current **stable** release is promoted to **lts**. This is not recommended for all AMP partners, but is provided so partners performing a QA cycle may do so less often. The **lts** release must be explicitly opted into by adding `/lts` to the path of the runtime and extension scripts requested in the page:
+
+> Note: The LTS release channel is available only to AMP HTML pages; it is not available to AMP Actions, AMP for Email, AMP Stories, or AMP for Ads.
+
+```html
+<!-- Standard AMP HTML runtime and extension scripts -->
+<script async src="https://cdn.ampproject.org/v0.js"></script>
+<script async custom-element='amp-ad'
+     src='https://cdn.ampproject.org/v0/amp-ad-0.1.js'></script>
+
+<!-- LTS AMP HTML runtime and extension scripts -->
+<script async src="https://cdn.ampproject.org/lts/v0.js"></script>
+<script async custom-element='amp-ad'
+     src='https://cdn.ampproject.org/lts/v0/amp-ad-0.1.js'></script>
+```
+
+As long as all runtime and extension scripts in a page must use the same release version, requesting the LTS scripts is considered valid AMP and gets the same cache benefits as the **stable** release channel.
+
+One trade-off to be aware of is that, because the **lts** release is updated every four weeks, the release binaries may be as much as six weeks behind the `HEAD` of [`ampproject/amphtml`](https://github.com/ampproject/amphtml). For this reason, partners using the **lts** release channel should not use very newly introduced features. See the section on [determining if your change is in a release](#Determining-if-your-change-is-in-a-release).
```

Comment:
~~very~~ newly

Reply:
whoops, started out with "very new", changed to "newly", kept both :laughing:

Reply:
Fixed.